new/LOC7435MayKwak.xml

```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7435MayKwak" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Māy Kʷāḵ</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-01-24">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="qushat">
+                    <placeName xml:id="n1" xml:lang="ti">ማይ፡ ኳክ፡</placeName>
+                    <placeName corresp="#n1" xml:lang="en">Māy Kʷāḵ</placeName> 
+                    <region notAfter="1991" ref="LOC6569Tegray" type="province"/>                    
+                    <settlement when="2024-01-09" ref="LOC7434HaddushAddi" type="tabiya">ṭābiyā Ḥādduš ʿAddi</settlement>
+                    <country ref="LOC3010Ethiop"></country>
+                </place>
+            </listPlace>
+            <listRelation>                        
+                <relation name="skos:exactMatch" active="LOC7435MayKwak" passive="INS1008EndaYohannes"/>
```

Comment:
it seems to me that `qushat `cannot be identical to `INS`, only a `church `/ `monastery `can be. if LOC7435 is the district then you can locate INS1008 in the district. 
You may (you do not have to) create a `LOC `for the church too, then these would be linked by the `exactMatch `relation

Reply:
I did it after `<relation name="skos:exactMatch" active="INS1007Taqot_Maryam" passive="LOC7409Taqot"/>`. Isn't Enda Yohannes an institution? It stands in the place (qushet) May Kwak, but somehow this name is not attached to the name of the church.

Reply:
Here, there is no qushet Enda Yohannes.

Reply:
District is a district, church is a church, they are not identical. We need exactMatch only if we have a LOC record for the church itself, and the church is also a repository and thus an INS. Then the very same things can be linked by exactMatch.

Reply:
Remove the `relation `here and under Enda Yohannes. It is enough that the qushat is under `settlement `there